jetbrains-core-gui/tst/software/aws/toolkits/jetbrains/settings/SetSamCli.kt

```diff
@@ -13,12 +13,23 @@ import com.intellij.testGuiFramework.impl.textfield
 import com.intellij.testGuiFramework.util.step
 import com.intellij.ui.SearchTextField
 import org.junit.Test
+import software.aws.toolkits.jetbrains.core.executables.ExecutableInstance
+import software.aws.toolkits.jetbrains.core.executables.ExecutableManager
+import software.aws.toolkits.jetbrains.core.executables.getExecutable
 import software.aws.toolkits.jetbrains.fixtures.openSettingsDialog
+import software.aws.toolkits.jetbrains.services.lambda.sam.SamExecutable
 
 class SetSamCli : GuiTestCase() {
     @Test
     fun setSamCli() {
-        val samPath = System.getenv("SAM_CLI_EXEC") ?: SamExecutableDetector().find() ?: "sam"
+        val instance = ExecutableManager.getInstance().getExecutable<SamExecutable>().thenApply {
+            if (it is ExecutableInstance.Executable) {
+                it
+            } else {
+                null
+            }
+        }.toCompletableFuture().join()
+        val samPath = System.getenv("SAM_CLI_EXEC") ?: instance?.executablePath.toString() ?: "sam"
```

Comment:
`instance?.executablePath?.toString()`?

Reply:
This was due to the revert not being 100% done, fixed